packages/e2e-tests/src/specs/editor/prepublishChecklist.js

```diff
@@ -51,4 +53,27 @@ describe('Pre-Publish Checklist', () => {
     await expect(page).toMatch('Add poster image');
     await percySnapshot(page, 'Prepublish checklist');
   });
+
+  describe('Contributor User', () => {
+    withUser('contributor', 'password');
+
+    it('should not let me publish a story as a contributor', async () => {
+      await createNewStory();
+
+      await insertStoryTitle('Publishing Flow: Contributor');
+
+      await expect(page).toClick('button[aria-label="Checklist"]');
+      await expect(page).toMatchElement(
+        '#pre-publish-checklist[data-isexpanded="true"]'
+      );
+
+      // verify no issues are present
+      await expect(page).toMatchElement('p', {
+        text: 'You are all set for now. Return to this checklist as you build your Web Story for tips on how to improve it.',
+      });
+
+      // verify that publish button is disabled
+      await page.$('button:disabled', { text: 'Publish' });
```

Comment:
```suggestion
      await expect(page).toMatchElement('button:disabled', { text: 'Publish' });
```

This needs an `expect` so that the test would fail if the element isn't found

Reply:
oh oops 🤦 good catch